backend/Makefile

```diff
@@ -63,12 +64,21 @@ image_launcher:
 
 .PHONY: dev-kind-cluster
 dev-kind-cluster:
+	${CONTAINER_ENGINE} build -t ${IMG_TAG_WEBHOOK_PROXY} -f $(CURDIR)/../tools/kind/Dockerfile.webhook-proxy $(CURDIR)/../tools/kind
 	-kind create cluster --name $(KIND_NAME) --config $(CURDIR)/../tools/kind/kind-config.yaml
 	kubectl config use-context kind-$(KIND_NAME)
 	kind get kubeconfig --name $(KIND_NAME) > $(CURDIR)/../kubeconfig_$(KIND_NAME)
+	kind --name $(KIND_NAME) load docker-image ${IMG_TAG_WEBHOOK_PROXY}
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.2/cert-manager.yaml
+	kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=180s
+	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=180s
+	$(MAKE) -C $(CURDIR)/src/crd/kubernetes install
 	kubectl apply -k $(CURDIR)/../manifests/kustomize/cluster-scoped-resources
 	kubectl wait --for condition=established --timeout=1m crd/applications.app.k8s.io
 	kubectl apply -k $(CURDIR)/../manifests/kustomize/env/dev-kind
+	kubectl apply -k $(CURDIR)/../manifests/kustomize/base/crds
+	kubectl apply -k $(CURDIR)/../manifests/kustomize/base/webhook
+	kubectl apply -f $(CURDIR)/../tools/kind/webhook-proxy.yaml
```

Comment:
Could you document this in the `backend/README.md` `API Server Development` section? It's not obvious to new developers that the webhook proxy is a workaround for local development API server not serving over HTTPS which Kubernetes requires.